projects/plugins/jetpack/extensions/blocks/send-a-message/block.json

```diff
@@ -0,0 +1,17 @@
+{
```

Comment:
I don't know what we should do with this one. Could we just surface the WhatsApp block instead?

Reply:
Hmm, What do you mean by surfacing?

Reply:
You mean exposing only data from `whatsapp-button`?